collection-scripts/gather_network_observability

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NO_NS="$(oc get subs -A -o template --template '{{range .items}}{{if eq .spec.name "netobserv-operator"}}{{.metadata.namespace}}{{end}}{{end}}')"
+
+if [ -z "${NO_NS}" ]; then
+    echo "INFO: Network Observability namespace is not detected. Skipping."
+    exit 0
+fi
+
+function get_netobserv_cluster_crs() {
+    declare -a NETOBSERV_CRDS=("flowcollectors")
+    
+    for CRD in "${NETOBSERV_CRDS[@]}"; do
+        oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" "${CRD}"
+    done
+}
+
+oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" "ns/${NO_NS}"
+oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" "ns/netobserv"
```

Comment:
"netobserv" namespace is configurable in the CRD. Can we extract that info from FlowCollector (`spec.namespace`) and inject it in this command ?